pkg/client/client.go

```diff
@@ -198,3 +198,18 @@ func (c *Client) ReadEntityWildcard(ctx context.Context, entity *p4_v1.Entity, r
 	}
 	return nil
 }
+
+func (c *Client) SendMessage(ctx context.Context, msg *p4_v1.StreamMessageRequest) error {
```

Comment:
please update https://github.com/antoninbas/p4runtime-go-client/blob/54febd69c5c7b89acf43072045f6e5d9a7f05032/pkg/client/digests.go#L9 to use `SendMessage`

Reply:
Done